versioneer.py

```diff
@@ -0,0 +1,1822 @@
+
```

Comment:
Should we add some statement or explanation here for the versioneer code?
And also the copyright of MONAI.

Reply:
this file is auto generated by versioneer and I feel it's already well documented